dashboard/src/app/colors/che-colors.styl

```diff
@@ -24,6 +24,13 @@ $frame-subscription-bg-color = alpha($che-medium-blue-color, 0.2)
 $auth-error-color = $che-orange-color
 $auth-ok-color = $che-green-color
 
+$list-blue-color = $che-medium-blue-color
```

Comment:
$list-blue-color - let the property describe part of list it is used for (border, selection, background, etc)